hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/util/Util.java

```diff
@@ -203,6 +205,21 @@ public static void checkSerializable(Object object, String objectName) {
         }
     }
 
+    /** Returns a deep clone of an object by serializing and deserializing it (ser-de). */
+    public static <T> T serde(T object) {
```

Comment:
shouldn't be this constrained to `Serializable` ? How will it work with classes loaded from job class loader

Reply:
This method is called when creating the DAG, not when running the job. It's called on the job submitter, and it has the classes on its classpath, so there should not be a problem